ansible/inventory/group_vars/apps.yaml

```diff
@@ -15,10 +15,12 @@ aem:
   deployment_check_delay_in_seconds: 5
   publish:
     jvm_mem_opts: "-Xss4m -Xms4096m -Xmx8192m"
+    jvm_opts: "-nofork"
```

Comment:
Please move this to examples.
-nofork is a specific scenario, most users don't need it, so it shouldn't be a default.

Reply:
Done, moved to examples for both full-set and consolidated